From a7a737a5f1bebc2a88b8247cb8c57bbf118ec8d6 Mon Sep 17 00:00:00 2001
From: Ovidiu Panait <ovidiu.panait@windriver.com>
Date: Thu, 24 Sep 2020 14:31:20 +0300
Subject: [PATCH 2/2] microblaze: spl: Change default boot device to RAM

Signed-off-by: Ovidiu Panait <ovidiu.panait@windriver.com>
---
 arch/microblaze/cpu/spl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/microblaze/cpu/spl.c b/arch/microblaze/cpu/spl.c
index 86522f8447..c2518463d1 100644
--- a/arch/microblaze/cpu/spl.c
+++ b/arch/microblaze/cpu/spl.c
@@ -17,7 +17,7 @@ bool boot_linux;
 
 u32 spl_boot_device(void)
 {
-	return BOOT_DEVICE_NOR;
+	return BOOT_DEVICE_RAM;
 }
 
 /* Board initialization after bss clearance */
-- 
2.25.1

